code/game/machinery/doors/door.dm

```diff
@@ -428,4 +428,9 @@
 	if(. && !(machine_stat & NOPOWER))
 		autoclose_in(DOOR_CLOSE_WAIT)
 
+/obj/machinery/door/zap_act(power, zap_flags)
+	zap_flags &= ~ZAP_OBJ_DAMAGE
```

Comment:
will this make the zaps chained off of a door also not damage objects?

Reply:
no, zap flags is a bitflag, so changing it in zap_act does nothing. This wasn't always the case, but it is now.